### DISCUSSION TITLE
DRAM, SRAM 차이

### DISCUSSION THREAD

**Discussion by 3Juhwan** [2024-woowacourse-study/level-interview#263] [Computer Architecture]
.

**Comment by le2sky** [comment]
RAM의 종류는 크게 DRAM, SRAM, SDRAM, DDR SDRAM이 존재한다. SDRAM과 DDR SDRAM은 DRAM 기반이므로 DRAM과 SRAM을 기준으로 비교한다. 

**DRAM(Dynamic RAM)**

DRAM은 저장된 데이터가 동적으로 변하는 RAM을 의미한다.  DRAM은 시간이 지나면 저장된 데이터가 사라진다. 따라서 DRAM은 데이터 소멸을 막기 위해서 일정 주기로 데이터를 재활성화 해야한다. 
- 소비 전력이 비교적 낮고 저렴하다.
- 주로 주기억장치로 사용된다.
- 집적도가 높기 때문에 대용량으로 설계하기 용이하다. 
- 집적도가 높다는 것은 더 작고 빽빽하게 만들 수 있다는 말 
    - ex) DRAM이 SRAM에 비해 집적도가 4배 크다면?
    - ex) 1비트 저장 공간을 만드는데, 칩의 공간 혹은 사용되는 트랜지스터 수가 적음
    - ex) DRAM 1bit = 트랜지스터 1개, SDRAM 1bit = 트랜지스터 4개

**SRAM(Static RAM)**

SRAM은 저장된 데이터가 변하지 않는 RAM을 의미한다. SRAM은 시간이 지나도 저장된 데이터가 사라지지 않는다. 따라서 SRAM은 주기적으로 데이터를 재활성화할 필요도 없다.
- 소비전력이 비교적 높고 비싸다.
- 집적도가 낮기 때문에 대용량으로 설계하는데 적합하지 않다.
- 상대적으로 빠르기 때문에 캐시 메모리로 활용된다.
